CHANGELOG.md

```diff
@@ -82,6 +82,12 @@ Note that 3.1.0 will be the first API stable release and interfaces in this rele
   * The configuration element `seqan3::search_cfg::mode` does not exist anymore.
     You can replace it by directly using one of the above mentioned "hit strategy" configuration elements
     ([\#1639](https://github.com/seqan/seqan3/pull/1639)).
+    
+#### Core
+
+* Configuration refactoring:
+  * The `seqan3::pipeable_config_element` has an additional template value that specifies whether a `value` member is 
+    present and in use. If a config element uses the `value` member, you need to set the 3rd template argument `true`.
```

Comment:
I think the configuration implementation is rather detail and is only for during the refactoring, so it shouldn't be in the changelog

Reply:
To me it looks like a public API change... However, I'll delete it.

Reply:
This is internal and is thus not a API change :)

Reply:
It's removed :)